lstchain/reco/volume_reducer.py

```diff
@@ -0,0 +1,145 @@
+"""
+**TEMPORTAL** module with functions to perform a volume reduction to LST data:
+
+NOTE : The volume reduction SHOULD BE performed at the dl0 data levels.
```

Comment:
"Applied at xxx level" for this method is unclear imo.
I would simply write "the volume reduction produces reduced waveforms and write them in the DL0 container"

Reply:
And maybe indicate that the event must have been calibrated before